_legacy/SnapshotScaffoldingProvider.php

```diff
@@ -3,14 +3,17 @@
 
 namespace SilverStripe\SnapshotAdmin;
 
+use SilverStripe\Admin\GraphQL\ReadOneLegacyResolver;
 use SilverStripe\CMS\Model\SiteTree;
 use SilverStripe\Core\ClassInfo;
 use SilverStripe\GraphQL\Scaffolding\Interfaces\ScaffoldingProvider;
+use SilverStripe\GraphQL\Scaffolding\Scaffolders\CRUD\ReadOne;
 use SilverStripe\GraphQL\Scaffolding\Scaffolders\SchemaScaffolder;
+use SilverStripe\GraphQL\Scaffolding\StaticSchema;
 use SilverStripe\ORM\DataObject;
 use SilverStripe\Security\Member;
 
-if (!interface_exists(ScaffoldingProvider::class)) {
+if (!interface_exists(ReadOneLegacyResolver::class) && !class_exists(ReadOneLegacyResolver::class)) {
```

Comment:
Does this need to check for the `ScaffoldingProvider` as it was before, and add the check for the `ReadOneLegacyResolver` class, instead of removing the check for the original interface completely?

Reply:
This is a typo, I will fix PR.